python/cloudtik/providers/_private/aliyun/node_provider.py

```diff
@@ -243,12 +240,13 @@ def create_node(
 
         created_nodes_dict = {}
         if count > 0:
-            filter_tags.append(
-                {"Key": CLOUDTIK_TAG_NODE_STATUS, "Value": tags[CLOUDTIK_TAG_NODE_STATUS]}
+            aliyun_format_tags = [{"Key": key, "Value": value} for key, value in tags.items()]
```

Comment:
The name aliyun_format_tags,  suggest to use the name meaning for current purpose, for example, tags_for_creation, instead for names like aliyun_format_tags